test/unit_tests/unit_test_math.cpp

```diff
@@ -114,6 +114,56 @@ namespace pvmath
         }
         return true;
     }
+
+    template <typename _Type>
+    bool pointProjection()
+    {
+        PointBase2D<_Type> point1( 0, 0 );
```

Comment:
Hi @fullset is it possible to create a random based values in a loop instead of hardcoded values?

Reply:
I guess it's possible. I just need to find out a good converting way from coordinates to projections.

Reply:
It's easy: you could assume that your line is vertical or horizontal and then third point is somehow locates at orthogonal line (vertical or horizontal) or at the same line.

Reply:
Oh, you're right. I was thinking about a universal formula.

Reply:
I guess I've done it.